src/types.ts

```diff
@@ -120,7 +120,7 @@ type RPCRequestParams<T extends Record<string, JSONValue> = ObjectEmpty> = {
     authorization: string;
     timeout: number;
   }>;
-};
+} & Omit <T, 'metadata'>;
```

Comment:
Don't you still need this?

Reply:
Yea we added it back in, it was causing issues before.